stable/nextcloud/README.md

```diff
@@ -150,6 +150,14 @@ The following table lists the configurable parameters of the nextcloud chart and
 | `hpa.maxPods`                                                | Max. pods for the Nextcloud HorizontalPodAutoscaler     | `10`                                        |
 | `deploymentAnnotations`                                      | Annotations to be added at 'deployment' level           | not set                                     |
 | `podAnnotations`                                             | Annotations to be added at 'pod' level                  | not set                                     |
+| `metrics.enabled`                                            | Start a single Prometheus metrics exporter              | `false`                                     |
+| `metrics.https`                                              | Wether https is used for scraping Nextcloud metrics     | `false`                                     |
```

Comment:
This is a bit misleading (at least to me)? The value sounds like you enable https on the exporter
While this defines if https or http is used to connect to nextcloud, right?

Reply:
Yes, you're right it's about the connection between nextcloud and the exporter. The exporter unfortunately needs this explicitly. I'm also not so happy with the naming. Do you have a suggestion for the name?